src/Centreon/Infrastructure/Monitoring/MonitoringRepositoryRDB.php

```diff
@@ -613,7 +613,7 @@ public function findServicesByIdsForAdminUser(array $serviceIds): array
                 $services[] = $service;
             }
         } else {
-            throw new \Exception(_('Bad SQL request'));
+            throw new \Exception(_('Failed to find services'));
```

Comment:
No need to throw an exception. If an exception is thrown by PDO it is up to the service to handle the case.

Reply:
indeed

Reply:
done :heavy_check_mark: